### ISSUE TITLE
Using longitudeFormatter in Hovmoller diagram

### ISSUE THREAD

**Issue by oliviermarti** [SciTools/cartopy#1478]
### Description

Hovmoller diagram are latitude-time or longitude-time diagrams.

As they are not on lat/lon axes, we cannot use Cartopy. We simply use Matplotlib.

But this prevent use to use `mpl.ticker.longitudeFormatter` to nicely format the geographical axis.

They is a exemple here : https://unidata.github.io/python-gallery/examples/Hovmoller_Diagram.html
The formatting of the longitude axis is complicated, and not very versatile when you change the region.

Is there a way to use `cartopy.mpl.ticker.LongitudeFormatter` for this type of plot ?

Thanks,

Olivier

**Comment by stefraynaud** [comment]
Olivier,

a workaround is to declare a simplified version of the formatters, like this:
```python
import matplotlib.pyplot as plt
import matplotlib.ticker as ticker
from cartopy.mpl.ticker import LongitudeFormatter

class MyLongitudeFormatter(LongitudeFormatter):
    def __call__(self, value, pos=None):
        return self._format_value(value, value)

ax = plt.subplot(111)
ax.set_xlim(0, 5)
ax.xaxis.set_major_formatter(MyLongitudeFormatter(dms=True))
```
![ticks](https://user-images.githubusercontent.com/1941408/75259776-d96c3d00-57e8-11ea-80ab-ba4dd7973673.png)

**Comment by stefraynaud** [comment]
To the admins: the formatters should probably be able to swith to this simplified mode if axes are not GeoAxes, instead of raise an error?

**Comment by dopplershift** [comment]
@stefraynaud Is this closed by #1482?

**Comment by stefraynaud** [comment]
Yes, it is!
